src/tests/system/data/test_passkey/readme.md

```diff
@@ -0,0 +1,56 @@
+Passkey support is tested with su, tests are running with
+umockdev, not with a physical key.
+
+First step is register the key
+the system administrator uses sssctl. 
+The output of this command can be set to the LDAP server in a dedicated attribute.
+    
+`sssctl passkey-register --username=<username> --domain=<domain name>`
+Next, it will ask for PIN and generate the passkey-mapping and token.
```

Comment:
Extra line,  the newline isn't showing up when it's formatted? 

https://github.com/SSSD/sssd/blob/f1f123c7c132eabc685309fad9e61566bdf3c4e0/src/tests/system/data/test_passkey/readme.md

Reply:
Weird because it's showing up in the comment correctly. 

`sssctl passkey-register --username=<username> --domain=<domain name>`
Next, it will ask for PIN and generate the passkey-mapping and token.